modules/mqtt/destination/mqtt-destination.c

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (c) 2021 One Identity
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#include "mqtt-destination.h"
+#include "mqtt-worker.h"
+#include "mqtt-parser.h"
+
+#include "plugin.h"
+#include "messages.h"
+#include "misc.h"
+#include "stats/stats-registry.h"
+#include "logqueue.h"
+#include "driver.h"
+#include "plugin-types.h"
+#include "logthrdest/logthrdestdrv.h"
+
+/*
+ * Default values
+ */
+
+#define DEFAULT_HOST "localhost"
+#define DEFAULT_PORT 1833
+#define DEFAULT_CLEAN_SESSION FALSE
+#define DEFAULT_KEEPALIVE 60
+#define DEFAULT_QOS 0
+#define DEFAULT_PROTOCOL_TYPE "tcp"
+#define DEFAULT_MESSAGE_TEMPLATE "$MESSAGE"
+
+
+/*
+ * Configuration
+ */
+
+void
+mqtt_destination_dd_set_host (LogDriver *d, const gchar *host)
```

Comment:
the prefix here is rendundant, "dd" means destination driver. So either "dd" or destination but not both.

We are usually using "dd" only. That makes the idientifiers a lot shorter.

Reply:
Solved by this commit: MQTT/destination: remove destination prefix
(commit hash: 508a8b2870ecf42ca84dcb7aa78acc30c299d746 )